lib/conversions.rb

```diff
@@ -1,9 +1,29 @@
 module Conversions
   module Distance
+    METRES_IN_MILE = 1609.344
+
     module Miles
       module ToMetres
         def self.convert(miles)
-          miles * 1609.344
+          miles * METRES_IN_MILE
+        end
+      end
+    end
+
+    module Metres
+      module ToMiles
+        def self.convert(metres, precision = 1)
```

Comment:
should this have a guard clause to return 0 if 0 meters is given?

Reply:
Good catch!

Reply:
Just tested, not needed.

returns 0 for 0 / 3 and 0.0 for 0.0 / 3